pages/CreatingAnExploration.html

```diff
@@ -9,20 +9,22 @@
 
   <p>
     After you've played an exploration, why not try your hand at making one? To
-    do so, navigate to the gallery, and look for the "Create Exploration"
+    do so, navigate to the gallery, and look for the "Create"
     button in the top navbar.
   </p>
 
   <img src="static/images/galleryCreateButton.png" style="width: 100%;">
 
   <p>
-    This will open a dialog box.
+    This will take you to the exploration editor page, click on the Setting icon on the top right 
+    corner of the page; this will take you to a page where in you fill the basic settings 
```

Comment:
... page. This will take you to a page where you can fill out some basic settings ...